api/chaos/serverexperimentation/v1/serverexperimentation.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+
+package clutch.chaos.serverexperimentation.v1;
+
+import "google/api/annotations.proto";
+import "validate/validate.proto";
+
+import "api/v1/annotations.proto";
+
+option go_package = "serverexperimentationv1";
+
+message TestConfig {
+  oneof target {
+    ClusterPairTarget cluster_pair = 1;
+  }
+
+  oneof fault {
```

Comment:
ideally, we should get rid of this oneof (now that the config is described as an any in Experiment). We can do this in a separate change, though